can/interfaces/pcan/pcan.py

```diff
@@ -83,6 +88,13 @@ def __init__(self, channel, *args, **kwargs):
         if result != PCAN_ERROR_OK:
             raise PcanError(self._get_formatted_error(result))
 
+        if win32event:
+            self._recv_event = win32event.CreateEvent(None, 0, 0, None)
+            result = self.m_objPCANBasic.SetValue(
```

Comment:
Unfortunately this is a Windows API thing. Would be nice if Peak could provide a more general way like a blocking function or callbacks like many other interfaces have. pywin32 is quite common on Windows installations but I should definitely add it to the installation instructions.